apps/api/src/app/shared/dtos/step-filter.ts

```diff
@@ -0,0 +1,98 @@
+import { BuilderFieldOperator, BuilderFieldType, BuilderGroupValues, FilterPartType } from '@novu/shared';
```

Comment:
Love this dto file!

Should we add .dto to the name? (to step-filter.dto.ts)

Reply:
it's in the `dto` folder, and other files there don't have the suffix `*.dto.ts`, that's why I named it like this